test/remote/gateways/remote_adyen_test.rb

```diff
@@ -605,6 +605,13 @@ def test_blank_country_for_purchase
     assert_match Gateway::STANDARD_ERROR_CODE[:incorrect_address], response.error_code
   end
 
+  def test_nil_state_for_purchase
+    @options[:billing_address][:state] = nil
+    response = @gateway.authorize(@amount, @credit_card, @options)
+    assert_failure response
+    assert_match Gateway::STANDARD_ERROR_CODE[:incorrect_address], response.error_code
+  end
+
```

Comment:
🤔 Wait a minute. Don't we want both these state tests to succeed? Are we sure N/A is what Adyen wants here?

Reply:
*facepalm* yes I suppose we do want that

Reply:
I'm curious. How does this just work now? Did you have to change an account setting or something?

Reply:
Hmm, the test has been flipped to expect success but the update doesn't seem to be self-aware in Github 🤔